src/Command/Api/ApiCommandBase.php

```diff
@@ -91,6 +91,10 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     }
 
     $path = $this->getRequestPath($input);
+    $acquia_cloud_client->addOption('headers', [
+      'Accept' => 'application/json',
```

Comment:
I don't know if this will work as expected, it might clobber the existing `User-Agent` header.

Reply:
I confirmed via server logs that the user agent is still applied as before.